docs/source/en/model_doc/dinov2.md

```diff
@@ -109,6 +109,36 @@ predicted_class_idx = logits.argmax(-1).item()
 print("Predicted class:", model.config.id2label[predicted_class_idx])
 ```
 
+The example below shows how to split the output tensor into one embedding for the whole image, i.e. the CLS token, and patch embeddings.
```

Comment:
Let's add this to the `## Notes` section.

```suggestion
The example below shows how to split the output tensor into one embedding for the whole image, for example, the `CLS` token and patch embeddings.
```

Reply:
I wouldn't translate "i.e." as "for example", it's doesn't have the same meaning in this context.

I tried to expand the text a bit more, remove ambiguity and abbreviations. Please have a look at the next commit to see if it reads better.

Reply:
Looks good! Remember to add it to the `## Notes` section :)

Reply:
Ok moving to `## Notes`